src/main/java/com/google/opensesame/servlets/MentorEntity.java

```diff
@@ -0,0 +1,21 @@
+package com.google.opensesame.servlets;
+
+import com.googlecode.objectify.annotation.Subclass;
+import java.util.ArrayList;
+
+@Subclass(index = true)
+public class MentorEntity extends PersonEntity {
+  ArrayList<Long> projectIDs;
+  ArrayList<String> menteeIDs;
+
+  private MentorEntity() {
+    super();
+  }
+
+  public MentorEntity(
+      PersonEntity person, ArrayList<Long> projectIDs, ArrayList<String> menteeIDs) {
+    super(person.id, person.gitHubId, person.interestTags, person.email);
+    this.projectIDs = projectIDs;
+    this.menteeIDs = menteeIDs;
```

Comment:
The comment I made about deprecating `PersonObject` also applies to `MentorObject`.